archive/dto.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: dto
```

Comment:
Wouldn't `android_dto` be a more appropriate name, when we already have https://formats.kaitai.io/android_img/index.html?

Reply:
I feel like it may be not Android-specific. https://github.com/devicetree-org/devicetree-specification/tree/master/source